Source/Analytics/Web.Frontend/src/components/HeaderPanel.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import { Heading, UnorderedList, ListItem } from "evergreen-ui";
+
+export class HeaderPanel extends React.Component {
+
+    render() {
+        let panelStyle = {
+            marginRight: 30
+        };
+
+        return (
+            <div className="analytics--headerPanel" style={panelStyle}>
+                <i className={`${this.props.icon} fa fa-5x analytics--headerPanelIcon`} />
+
+                <Heading color={this.props.color} size={600} paddingTop={"20px"}>
+                    {this.props.headline}
+                </Heading>
+
+                <div className="analytics--listContainer">
+                    <UnorderedList size={500}>
+                        {this.props.list.map(
+                            (data, index) => (
+                                <ListItem key={index}>
+                                    {data.numberOfReports || data.inactiveDataCollectors} {data.name || data.healthRisk}
```

Comment:
Shouldn't rather the data that we send in be shaped so we have the correct format of it, instead of "forcing" this component to be linked to the datastructure (and name of properties) of the input data?

Reply:
Agreed. I rewrote it, now it takes any JSON Object and lists them "value-first" no matter the keys:

`
{name: "Cholera", numberOfReports: "20"} --> "20 Cholera"
`

Not very scalable, but compatible with the current API endpoints. Could be a good idea to include this as a test in a frontend testing framework in the future.